spinta/types/model.py

```diff
@@ -69,6 +69,15 @@ def check(context: Context, model: Model):
         context.deprecation("`id` property must specify real type like 'string' or 'integer'. Use of 'pk' is deprecated.")
 
 
+@check.register()
+def check(context: Context, model: Model, data: dict):
+    for name, prop in model.properties.items():
+        if name in data:
+            data_value = data[name]
+            if not prop.type.is_valid(data_value):
+                context.error(f"{data_value} is not valid type: {prop.type}")
```

Comment:
`context.error` is an old leftover, don't use it. Probably here it is better to raise `ValidationError` or something like that, then you should handle these check errors and return `400` response.